examples/nextjs-with-typescript/styles.css

```diff
@@ -34,8 +34,8 @@ header {
 
 header h1 {
   margin: 0;
-  margin-left: .9em;
-  margin-right: .9em;
+  margin-left: 0.9em;
```

Comment:
Apologies for the additional changes. These were all prettier rules, applied so I could actually get auto-formatting for the tsx. Actual style changes should be obvious.